client/signup/wpcom-login-form/index.jsx

```diff
@@ -1,13 +1,13 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	debug = require( 'debug' )( 'calypso:signup:wpcom-login' );
+import React from 'react';
+const debug = require( 'debug' )( 'calypso:signup:wpcom-login' );
```

Comment:
I think we usually use a different syntax for this:

```js
import debugFactory from 'debug'; // sometimes named debugModule
const debug = debugFactory( 'calypso:whatever' );
```

Per:

```
wp-calypso (master) ☯ spot client/ const debug | grep debug | grep require | wc -l
      15
wp-calypso (master) ☯ spot client/ const debug | grep debug | grep Module | wc -l
      41
wp-calypso (master) ☯ spot client/ const debug | grep debug | grep Factory | wc -l
      68
```

I don't see a reason to not use the shorter syntax you have here, but I thought it was worth mentioning.